index.html

```diff
@@ -358,7 +360,10 @@ <h4>Skills</h4>
       </section>
 
       <!-- Footer Section -->
-      <footer class="section footer-section bg-decoration">
+      <footer
+        id="#contact-section"
+        class="section footer-section bg-decoration"
```

Comment:
- [ ] It would be better if our navigation links in the hamburger redirects us to the contact page when the contact link is clicked. The contact page page is where we have the form
![Screenshot from 2022-09-06 15-58-37](https://user-images.githubusercontent.com/46388113/188684338-0a885dcc-08e8-46b0-8f7d-935bb507f1f5.png)

However, it takes us to this part instead as we have in the screenshot below

![Screenshot from 2022-09-06 15-59-23](https://user-images.githubusercontent.com/46388113/188684631-b5377826-2936-4bba-8ca9-da6941668edb.png)